tests/contest/contest/src/tests/process_user/process_user_test.rs

```diff
@@ -0,0 +1,39 @@
+use anyhow::{Context, Ok, Result};
+use oci_spec::runtime::{ProcessBuilder, Spec, SpecBuilder, UserBuilder};
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec() -> Result<Spec> {
+    let user = UserBuilder::default()
+        .uid(10u32)
+        .gid(10u32)
+        .additional_gids(vec![5u32])
```

Comment:
For this, can we do something like having 5 random gids with values in range 5..15 (maybe us ts as source of random?)

This is not strictly necessary, but I feel this would be good to have a more strict test of runtime logic.

Reply:
I set a random array. Is this ok?

https://github.com/youki-dev/youki/pull/2978/commits/31977e4a1c86b8570d9f8456ac1a202913eae0bb